readthedocs/templates/projects/project_edit_base.html

```diff
@@ -26,6 +26,7 @@
         <li class="{% block project-integrations-active %}{% endblock %}"><a href="{% url "projects_integrations" project.slug %}">{% trans "Integrations" %}</a></li>
         <li class="{% block project-environment-variables-active %}{% endblock %}"><a href="{% url "projects_environmentvariables" project.slug %}">{% trans "Environment Variables" %}</a></li>
         <li class="{% block project-automation-rules-active %}{% endblock %}"><a href="{% url "projects_automation_rule_list" project.slug %}">{% trans "Automation Rules" %}</a></li>
+        <li class="{% block project-webhooks-active %}{% endblock %}"><a href="{% url "projects_webhooks" project.slug %}">{% trans "Webhooks" %}</a></li>
         <li class="{% block project-notifications-active %}{% endblock %}"><a href="{% url "projects_notifications" project.slug %}">{% trans "Notifications" %}</a></li>
```

Comment:
Do we need another tab here? I think it will be confusing to have "Webhooks" and "Notifications -> Webhook"

At least, we should note this somewhere (e.g. ext-theme) to consider refactoring this in the new templates.

Reply:
I was thinking to rename this to "Email notifications" and webhooks would be moved out of this page.